aspnetcore/security/authentication/scaffold-identity.md

```diff
@@ -28,6 +40,124 @@ Services are required when using [Two Factor Authentication](xref:security/authe
 
 Typically, apps that were created with individual accounts should ***not*** create a new data context.
 
+## Scaffold Identity into a Blazor project
+
+*This section applies to Blazor Web Apps and Blazor Server apps.*
+
+Run the Identity scaffolder:
+
+# [Visual Studio](#tab/visual-studio)
+
+* From **Solution Explorer**, right-click on the project > **Add** > **New Scaffolded Item**.
+* From the left pane of the **Add New Scaffolded Item** dialog, select **Identity**. Select **Blazor Identity** in the center pane. Select the **Add** button.
+* In the **Add Blazor Identity** dialog:
+  * Select or add with the plus (**+**) button the database context class (**DbContext class**).
+  * Select the database provider (**Database provider**), which defaults to SQL Server.
+  * Select or add with the plus (**+**) button the user class (**User class**).
+  * Select the **Add** button.
+
+# [.NET CLI](#tab/net-cli)
+
+To add the required NuGet packages and tools, execute the following .NET CLI commands in a command shell opened to the project's root folder.
+
+Paste all of the following commands at the prompt (`>`) of the command shell. When you paste multiple commands, a warning appears stating that multiple commands will execute. Dismiss the warning and proceed with the paste operation.
```

Comment:
Do you dismiss or ignore? IIRC, there is nothing to dismiss. When I read "Dismiss", I'm thinking I must take action other.

Reply:
I can fix this ... the actual button text to get rid of the warning is **Paste anyway**, so I'll call that button out by name ...

```suggestion
Paste all of the following commands at the prompt (`>`) of the command shell. When you paste multiple commands, a warning appears stating that multiple commands will execute. Select the **Paste anyway** button.
```